packages/web-api-send-notification-runner/src/tasks/notification-sender-web-api-client.spec.ts

```diff
@@ -69,6 +69,7 @@ describe(NotificationSenderWebAPIClient, () => {
         const requestBody = { scanId: scanId, runStatus: runStatus, scanStatus: scanStatus };
         const options = {
             body: requestBody,
+            timeout: 5000,
```

Comment:
why do we need this? i don't think it will be stuck if the url doesnt respond. it should have a default timeout. what is the default timeout?

Reply:
since we use npm request package, so as mentioned in the timeouts section here:
https://www.npmjs.com/package/request
"Most requests to external servers should have a timeout attached, in case the server is not responding in a timely manner. Without a timeout, your code may have a socket open/consume resources for minutes or more."